core/shared/src/test/scala/com/softwaremill/sttp/UriTests.scala

```diff
@@ -22,12 +22,12 @@ class UriTests extends FunSuite with Matchers {
     Uri("http", None, "example.com", None, List(""), List(QF.KeyValue("p", "v"), QF.KeyValue("p", "v")), None) -> "http://example.com/?p=v&p=v",
     Uri("http",
         None,
-        "exa mple.com",
```

Comment:
should this be changed?

Reply:
The JavaScript `URL` class cannot parse that host name. As far as I know a hostname with spaces is not valid?

Reply:
Well if the whitespace is %-encoded, this URL is valid. So you are saying JavaScript can't parse the escaped version as well?

Reply:
Seems like Chrome is okay with it, but node isn't:
```
$ node
> new URL("http://example.com").host
'example.com'
> new URL("http://ex ample.com").host
TypeError [ERR_INVALID_URL]: Invalid URL: http://ex ample.com
    at onParseError (internal/url.js:234:17)
    at parse (internal/url.js:243:3)
    at new URL (internal/url.js:318:5)
```

```
new URL("http://example.com").host
20:23:41.772 "example.com"
20:23:45.796 new URL("http://ex ample.com").host
20:23:45.801 "ex%20ample.com"
```

I have taken the easy way out and just changed the tests to run in Chrome instead.